analytics_data_api/v0/views/learners.py

```diff
@@ -168,7 +181,7 @@ def _validate_query_params(self):
 
     def get_queryset(self):
         """
-        Fetches the user list from elasticsearch.  Note that an
+        Fetches the user list from elasticsearch. Note that an
```

Comment:
:( I actually use two-spaces in docstrings on-purpose. It 1) makes sentences easier to read when the editor uses a monospaced font (e.g. all code editors), and particularly 2) emacs recognizes sentences as delineated by two-spaces, so you can efficiently navigate by them.

Also, this docstring will not show up in the API docs.